newrelic/hooks/mlmodel_openai.py

```diff
@@ -432,8 +433,9 @@ async def wrap_embedding_acreate(wrapped, instance, args, kwargs):
 
     settings = transaction.settings if transaction.settings is not None else global_settings()
 
-    ft_name = callable_name(wrapped)
-    with FunctionTrace(ft_name) as ft:
+    function_name = wrapped.__name__
```

Comment:
With OpenAI v1, all functions are named `create` and there's no concept of `acreate` because the clients themselves become sync or async so we won't be able to distinguish between the two. It may be something we consider for the OpenAI v1 PRs.